src/interactions/interaction.ts

```diff
@@ -9,10 +9,52 @@ module Plottable {
      * "foreground" and "background" elements where it can draw things,
      * e.g. crosshairs.
      */
-    protected _componentToListenTo: Component;
+    protected _componentAttachedTo: Component;
```

Comment:
The commenting system here is pretty hard to follow. Pasting my argument here as well

Not suggestive enough without prior knowledge of how this class works. maybe `_componentAttachedTo` would work (doing this change)